website/docs/en-US/loading.md

```diff
@@ -167,6 +237,44 @@ Show a full screen animation while loading data.
     }
   }
 </script>
+<!--
+<setup>
+
+  import { defineComponent, ref, getCurrentInstance } from 'vue';
+
+  export default defineComponent({
+    setup() {
+      const { proxy } = getCurrentInstance();
+      const fullscreenLoading = ref(false);
+      const openFullScreen1 = () => {
+        fullscreenLoading.value = true;
+        setTimeout(() => {
+          fullscreenLoading.value = false;
+        }, 2000);
+      };
+
+      const openFullScreen2 = () => {
+        const loading = proxy.$loading({
```

Comment:
can we use 
```
import { ElLoading } from 'element-plus';
```
here, pls?

Reply:
It's so embarrassing. I was trying to fix it, but I didn't notice it. I will pay more attention to this part in the subsequent code modification

Reply:
That's not a problem, take your time. Thanks for your lovely contribution.

Reply:
have updated